examples/GLUE/src/train.py

```diff
@@ -0,0 +1,368 @@
+from pymarlin.core.module_interface import ModuleInterface
+from pymarlin.core.trainer import Trainer, TrainerArguments, WriterInitArguments,DistributedTrainingArguments,DefaultCheckpointerArguments
+from pymarlin.utils.stats.basic_stats import StatInitArguments
+from pymarlin.core.trainer_backend import SingleProcess,SingleProcessAmp, DDPTrainerBackend, SingleProcessApexAmp
+
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torch.optim.lr_scheduler import OneCycleLR
+
+from transformers import AutoModel, AutoTokenizer
+from transformers.modeling_outputs import SequenceClassifierOutput
+from pymarlin.utils.config_parser.custom_arg_parser import CustomArgParser
+from pymarlin.utils.stats import global_stats
+from sklearn.metrics import (
```

Comment:
*optional : could it be worth it to use metrics from the HF dataset instead of setting up the mapping here per glue_task? it would simplify the example a bit but you've already done the work.